frontend/src/components/NavBar/index.tsx

```diff
@@ -1,32 +1,51 @@
 import { useContext } from 'react'
+import { Button, Container, Nav, Navbar } from 'react-bootstrap'
+import { FaSignInAlt, FaUser, FaHome, FaSignOutAlt } from 'react-icons/fa'
 import { Link } from 'react-router-dom'
 
 import { AuthContext } from '../../context/AuthContext'
 import { CanAccess } from '../CanAccess'
 
 export function NavBar () {
-  const { isAuthenticated, user, signOut } = useContext(AuthContext)
+  const { isAuthenticated, signOut } = useContext(AuthContext)
 
   return (
-    <div>
-      <Link to="/login">Login</Link>
-      <Link to="/register">Register</Link>
-      <Link to="/">Home</Link>
+    <>
+    <Navbar bg='dark' variant='dark'>
+      <Container className='justify-content-center'>
+        <Nav>
 
-      <CanAccess permissions={['users.list']}>
-        <Link to="/users">Users</Link>
-      </CanAccess>
+        <Nav.Link as={Link} to="/"><FaHome /> Home</Nav.Link>
 
-      <CanAccess permissions={['metrics.list']}>
-        <Link to="/metrics">Metrics</Link>
-      </CanAccess>
+            {!isAuthenticated ? <Nav.Link as={Link} to="/register"><FaUser /> Register </Nav.Link> : <Nav.Link as={Link} to="/account"><FaUser /> Account </Nav.Link>}
+            {!isAuthenticated
+              ? <Nav.Link as={Link} to="/login"> <FaSignInAlt /> Login </Nav.Link>
+              : <Nav.Link as={Button} variant="dark" bg="dark" data-testid="logout-button"
+                onClick={() => signOut()}> <FaSignOutAlt /> Logout </Nav.Link>
+         }
 
-      {isAuthenticated && (
+        <CanAccess permissions={['users.list']}>
+        <Nav.Link as={Link} to="/users">Users</Nav.Link>
+        </CanAccess>
+
+        <CanAccess permissions={['metrics.list']}>
+        <Nav.Link as={Link} to="/metrics">Metrics</Nav.Link>
+        </CanAccess>
+
+      {/* {isAuthenticated && (
```

Comment:
@BumbleGoose when removing code, rather remove it completely instead of just commenting it out.